src/index.ts

```diff
@@ -401,16 +402,13 @@ export const typeORMDriver = {
           ok: (res: QueryResult) => void,
           fail: (msg: string) => void
         ) => {
-          try {
-            let response = QuickSQLite.execute(options.name, sql, params);
-            enhanceQueryResult(response);
-            ok(response);
-          } catch (e) {
-            fail(e);
-          }
+          QuickSQLite.executeAsync(options.name, sql, params)
+            .then(enhanceQueryResult)
+            .then(ok)
+            .catch(fail);
         },
-        transaction: (fn: (tx: Transaction) => void): void => {
```

Comment:
This is actually not used by TypeORM I think, this is left over from a wrong use of a wrapper